### DISCUSSION TITLE
trigger browser notification when starting a specific step + build 🔔

### DISCUSSION THREAD

**Discussion by sam-beautypie** [community/community#9243] [Actions]
As a developer, When i write code that is likely to break on CI, Then i want to be identified at the relevant point in the workflow, So i don't have to watch and wait 👀 .

Ideally there would be a small 🔔  icon next to each pending step in a workflow, which if pressed, triggered a browser notification when it starts and/or finished.

E.g. 
![image](https://user-images.githubusercontent.com/94920667/147215972-afb595d8-7ce1-4be5-9c83-056f6f4623a3.png)

I don't know how this would work for steps in an upcoming (dependent) jobs as i assume you don't know the steps up front (or at least they don't currently show).

**Comment by ethomson** [comment]
Interesting idea!  Thanks for the feedback - the team's a bit busy with some other things at the moment but we'll give this some thought for the future.
